slack.go

```diff
@@ -20,15 +20,17 @@ type Slack struct {
 func (s Slack) Notify(results []Result) error {
 	for _, result := range results {
 		if !result.Healthy {
-			s.Send(result)
+			notifications[result.Title] = 1
+			s.Send(result, "danger")
+		} else if notifications[result.Title] == 1 {
+			notifications[result.Title] = 0
+			s.Send(result, "good")
```

Comment:
While we're at it, I wonder if the colors should be moved into constants as well.